docs/content_management/configure_ct_field_settings.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Configure Field settings in Content Types.
+---
+
+# Configure Content Type Fields
+
+When you [modify a Content Type](create_edit_content_types.md) and add or modify Fields, 
+you must configure different settings that control the way [[= product_name =]] 
+treats the contents of each Field.
+
+You can only create or modify Content Types when your [user role](../permission_management/work_with_permissions.md) has the `ContentType/Create` or `ContentType/Update` permission.
+
+To modify Field settings, in the **Field definitions** section, expand the **Landing page** header. 
+When done, click **Save** to apply changes to the Content Type.
+ 
+## General settings
+
+Depending on their type, Fields can have different combinations of the following 
+general settings. 
+
+|Setting|Description|Use|
+--------|-----------|---|
+|Name|A user-friendly name that describes the Field, used in the interface. It can be up to 255 characters long and consist of letters, digits, spaces and special characters.|Required|
+|Identifier|An identifier for system use in configuration files, templates, or PHP code. It can be up to 50 characters long and can only contain lowercase letters, digits and underscores. Also used in name patterns for the Content Type.|Required|
+|Description|A detailed description of the Field. It is displayed next the it when the user edits the Content item.|Optional|
+|Required|Indicates whether a value the Field is required for the Content item to be saved or  published.|Optional|
+|Searchable|Indicates whether a value of the Field is included in the search.|Optional|
+|Translatable|Indicates whether a value of the Field can be translated.|Optional|
+|Can be a thumbnail|Indicates whether the Field can be a thumbnail.|Optional|
+
+!!! note
+
+    If the Field is required, it is recommended that you provide a default value for it.
+
+## Default configuration of Pages
```

Comment:
What about other Field Types ?

Reply:
This is the information I have at present. I am open to any suggestions you may have to add more FT-specific settings.

Reply:
At this point I added a statement that these two sections describe merely examples of FT configurations that the user may stumble upon. Wy may add more such examples when we have imput for them.